### ISSUE TITLE
bash: ./optool: cannot execute binary file: Exec format error

### ISSUE THREAD

**Issue by jmukes97** [CokePokes/PhantomExo#4]
Is there a specific operating system i should be using when trying to install this? I am using debian Linux (32 bit) and i keep getting the error above.

**Comment by Qadz** [comment]
Which IPA are you using ??

**Comment by jmukes97** [comment]
I am using an ipa that i downloaded from here:

http://www.filepup.net/get/7clJpZxk1475357547/1475524909/Snapchat__Snapchat_Inc.___v9.40.1_v9.40.1.0_PI_LP_os80_-Widow.rc334b_902.ipa

I am using an iphone 5 so the ipa link in the readme.md file wasn't compatable. I dont think it is a problem with the IPA though. It is the optool binary that isnt running.

**Comment by Qadz** [comment]
have you tired it with another operating system ? or another laptop ?

**Comment by August712** [comment]
MacOS only I think

**Comment by akshaypatil4321** [comment]
jmukes97 Can you please share the ipa file you have? Because I also have the iphone 5 and i am getting error of your iPhone is not in the supported device list.

**Comment by jmukes97** [comment]
I have tried it with a Mac and that seems like the only Mac will run the optool. I am still having trouble installing the ipa though. @akshaypatil4321 here is a link in my drive to the ipa

https://drive.google.com/file/d/0B2q-leawpBhxSzlMVG03cEZWdVk/view?usp=sharing

**Comment by akshaypatil4321** [comment]
someone said that it has to do with cydia impactor. According to him cydia impactor doesn't support iPhone 5.

**Comment by jmukes97** [comment]
But i have used cydia impactor to install other ipa files like Pokemon go ++. Are you sure?

**Comment by August712** [comment]
@akshaypatil4321 iPhone 5 works. It uses AFC which is on literally every iOS device. @jmukes97 yeah I thought only macs can run Mach-O binaries. And I'll assume that that error means that you downloaded an app that was cracked on a jailbroken 64-bit device using Rasticrac. Since your device is 32-bit, it doesn't have the correct architecture, and It can't run properly.

**Comment by August712** [comment]
In layman's terms, 64-bit devices can run 64-bit and 32-but binaries, but 32-bit devices can only run 32-bit binaries.

**Comment by August712** [comment]
Also sorry about taking forever to respond

**Comment by akshaypatil4321** [comment]
Can you please share cracked version of 32bit then with phantom?

**Comment by August712** [comment]
You'd need someone with the iOS 9.0.2 Jailbreak on an iPhone 5 to dump it for you. Which isn't me. Sorry.
